# ensure you have any custom patches applied before extending!
# created with:
# git diff -U1 -u Scripts/Common/Fix_CVE_Patchers.sh 

diff --git a/Scripts/Common/Fix_CVE_Patchers.sh b/Scripts/Common/Fix_CVE_Patchers.sh
index b2a21e1d8dd..1de8f610f6f 100644
--- a/Scripts/Common/Fix_CVE_Patchers.sh
+++ b/Scripts/Common/Fix_CVE_Patchers.sh
@@ -39,2 +39,3 @@ commentPatches() {
 #      CVE-2021-Misc2/3.4/0049.patch
+commentPatches android_kernel_samsung_exynos7870.sh "CVE-2019-16746/3.18/0006.patch" "CVE-2020-16166/3.18/0002.patch" "0006-AndroidHardening-Kernel_Hardening/3.18/0030.patch" "CVE-2020-0427/3.18/0007.patch"
 
@@ -72,3 +73,3 @@ commentPatches android_kernel_htc_msm8994.sh "CVE-2016-8394/ANY/0001.patch" "CVE
 commentPatches android_kernel_lge_bullhead.sh "CVE-2014-8559";
-commentPatches android_kernel_lge_msm8992.sh "CVE-2018-5897" "CVE-2018-11266";
+commentPatches android_kernel_lge_msm8992.sh "CVE-2018-5897" "CVE-2018-11266" "CVE-2018-5849";
 commentPatches android_kernel_lge_msm8996.sh "0008-Graphene-Kernel_Hardening-canaries/4.4/0002.patch";
@@ -132,3 +133,3 @@ done
 #3.18
-declare -a threeDotEighteen=("${threeDotTen[@]}" "android_kernel_samsung_universal8890.sh" "android_kernel_google_dragon.sh" "android_kernel_zte_msm8996.sh" "android_kernel_asus_msm8953.sh" "android_kernel_google_marlin.sh" "android_kernel_motorola_msm8996.sh" "android_kernel_oneplus_msm8996.sh");
+declare -a threeDotEighteen=("${threeDotTen[@]}" "android_kernel_samsung_universal8890.sh" "android_kernel_google_dragon.sh" "android_kernel_zte_msm8996.sh" "android_kernel_asus_msm8953.sh" "android_kernel_google_marlin.sh" "android_kernel_motorola_msm8996.sh" "android_kernel_oneplus_msm8996.sh" "android_kernel_samsung_exynos7870.sh");
 for script in "${threeDotEighteen[@]}"
